weekly-mission/next-project/components/Button/Button.jsx

```diff
@@ -0,0 +1,18 @@
+import styles from "./Button.module.css";
+
+const Button = ({ text, type, onClick }) => {
+  const buttonClass =
+    type === "primary" ? styles.primary : type === "red" ? styles.red : "";
```

Comment:
```suggestion
  const buttonClass = type ? styles[type] : ''
```

이런 식으로 쓰면 간단할 거 같아요~